utils/resource/usage.go

```diff
@@ -65,7 +65,6 @@ type Manager interface {
 	// Shutdown allocated resources and stop tracking all processes.
 	Shutdown()
 }
-
 type manager struct {
```

Comment:
Seems like an unrelated diff now
```suggestion

type manager struct {
```